runelite-client/src/main/java/net/runelite/client/plugins/itemstats/ItemStatChanges.java

```diff
@@ -169,21 +193,58 @@ private void init()
 		add(combo(3, boost(ATTACK, perc(.15, 5)), boost(STRENGTH, perc(.15, 5)), boost(DEFENCE, perc(.15, 5)), heal(HITPOINTS, -10)), DIVINE_SUPER_COMBAT_POTION1, DIVINE_SUPER_COMBAT_POTION2, DIVINE_SUPER_COMBAT_POTION3, DIVINE_SUPER_COMBAT_POTION4);
 		add(combo(2, boost(RANGED, perc(0.1, 4)), boost(DEFENCE, perc(0.15, 5)), heal(HITPOINTS, -10)), DIVINE_BASTION_POTION1, DIVINE_BASTION_POTION2, DIVINE_BASTION_POTION3, DIVINE_BASTION_POTION4);
 		add(combo(2, boost(MAGIC, 4), boost(DEFENCE, perc(0.15, 5)), heal(HITPOINTS, -10)), DIVINE_BATTLEMAGE_POTION1, DIVINE_BATTLEMAGE_POTION2, DIVINE_BATTLEMAGE_POTION3, DIVINE_BATTLEMAGE_POTION4);
+		add(combo(5, heal(PRAYER, perc(0.1, 2)), boost(MAGIC, perc(0.05, 2)), heal(ATTACK, perc(0.1, -2)), heal(STRENGTH, perc(0.1, -2)), heal(DEFENCE, perc(0.1, -2))),
+			ANCIENT_BREW4, ANCIENT_BREW3, ANCIENT_BREW2, ANCIENT_BREW1);
```

Comment:
This is not correct as it does not handle the maximum capped value of prayer (only boosts up to 5% above max). I would prefer to take #14566 for this, once it's been completed.